packages/package_info_plus/package_info_plus/example/windows/runner/flutter_window.cpp

```diff
@@ -4,7 +4,7 @@
 
 #include "flutter/generated_plugin_registrant.h"
 
-FlutterWindow::FlutterWindow(const flutter::DartProject& project)
+FlutterWindow::FlutterWindow(const flutter::DartProject &project)
```

Comment:
Was the windows cpp files wrong formatted before? or is CI not checking this?

Reply:
`melos format` did this. As I'm not familiar with cpp files, I trusted the tool.